### DISCUSSION TITLE
Hide definition in a prototype of the declaration section

### DISCUSSION THREAD

**Discussion by eepp** [doxygen/doxygen#11218] [Q&A]
Using Doxygen 1.12.0.

I have a C header file like this:

```
/*! @file */

/*!
Does something.

@param a The A.
*/
extern void meow_mix(int a) __MEOW;
```

`__MEOW` is defined by the build system.

I want to hide it from the documentation.

My `Doxyfile` is:

```
PROJECT_NAME           = test
OPTIMIZE_OUTPUT_FOR_C  = YES
HIDE_SCOPE_NAMES       = YES
GENERATE_LATEX         = NO
ENABLE_PREPROCESSING   = YES
PREDEFINED             = "__MEOW="
INPUT                  = test.h
```

Yet I see this:

![image](https://github.com/user-attachments/assets/01bc64b1-def0-4fd0-b110-cf45ed8f669a)

Is there a way to hide it?

**Comment by albert-github** [comment]
Please always mention the doxygen version used.

I think you need to set the setting `MACRO_EXPANSION = YES`

**Reply by eepp** [reply]
Updated with the version.

This works, thanks!
